cinema/urls.py

```diff
@@ -15,6 +16,7 @@
 router.register("cinema_halls", CinemaHallViewSet)
 router.register("movies", MovieViewSet)
 router.register("movie_sessions", MovieSessionViewSet)
+router.register("orders", OrderViewSet)
 
 urlpatterns = [path("", include(router.urls))]
```

Comment:
You can make it shorter urlpatterns = router.urls

Reply:
it's even not my code)